website/blog/2024-01-24-release-1.7.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Release Notes - Podman Desktop 1.7
+description: Podman Desktop 1.7 has been released!
+slug: podman-desktop-release-1.7
+authors: deboer
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.7/renovations.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.7 Release! 🎉
+
+![Podman-desktop-1-7-hero](img/podman-desktop-release-1.7/renovations.png)
+
+This release introduces:
+
+- **Podman 4.9.0**: [Podman 4.9.0](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Extension API Improvements**: A big update to the extension API enabling more goodness for 🦭 Podman Desktop's extensions.
+- **Experimental Kubernetes UI**: Get a sneak peek at new UI coming to help work with Kubernetes clusters.
+- **Enhanced Builds, Pods List, and Troubleshooting**: Build for different platforms, an upgraded pods view, and more.
+
+Podman Desktop 1.7 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Podman 4.9
+
+🦭 Podman 4.9 includes key fixes for stability and reliability problems reported by our users.
+If you've been floundering we highly recommend updating!
+
+### Extension API Improvements
+
+We spent a lot of time this release adding new extension API to give upcoming extensions
```

Comment:
```suggestion
We have spent a lot of time this release adding new extension API calls to give upcoming extensions
```

Reply:
Fixed the first in new commit. Didn't fix 'calls', because to me that sounds like the thing calling the API (ie extensions), not the interface (API) that we added here. Maybe there's another way to describe it, or it's good enough?